packages/fetchai/skills/ml_train/skill.yaml

```diff
@@ -19,7 +19,12 @@ handlers:
   transaction:
     class_name: MyTransactionHandler
     args: {}
-tasks: {}
+tasks:
```

Comment:
@marcofavorito I had to add this with to make it pass with your changes. Is that intentional?